dotCMS/src/integration-test/java/com/dotmarketing/util/HostUtilTest.java

```diff
@@ -0,0 +1,171 @@
+package com.dotmarketing.util;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.api.web.HttpServletRequestThreadLocal;
+import com.dotcms.datagen.SiteDataGen;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.UserAPI;
+import com.dotmarketing.portlets.contentlet.business.HostAPI;
+import com.liferay.portal.model.User;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class HostUtilTest extends IntegrationTestBase {
+
+    private static HostAPI hostAPI;
+    private static UserAPI userAPI;
+
+    private static User systemUser;
+    private static User testUser;
+    private static Host defaultHost;
+    private static String defaultHostId;
+    @BeforeClass
+    public static void prepare () throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4828838)